docker-compose.yaml

```diff
@@ -1,10 +1,27 @@
 version: '3.7'
```

Comment:
This morning talking about the move to non prod, we had issues with the registry argument not being passed from the pipeline when building the images.  This meant it was looking in the wrong registry.  No big deal to change the default argument in the Dockerfile for the purposes of testing.  But it would be better if the docker compose file accepted an argument for the registry which could be taken from the Jenkins environment variable.

Something like:

```
build:
      context: .
      args: 
        REGISTRY: ${DOCKER_REGISTRY:562955126301.dkr.ecr.eu-west-2.amazonaws.com}
```

Reply:
Good idea, I've set that to use an environment variable now. Added a `.env` file to set a default value, as that's referenced by docker-compose automatically.